app-demo/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
 }
 
 android {
+    namespace = "com.cardinalblue.demo"
```

Comment:
Would it be too general? How about `com.cardinalblue.gesture.demo`?

Reply:
Yeah. But it was like this, and the change is big for the main repo, so I don't want to change on the other thing for now.
Maybe I will be another PR after the update of the main repo is done.